aicsimageio/tests/readers/test_tiff_reader.py

```diff
@@ -26,6 +26,13 @@
             np.uint16,
             0,
         ),
+        (
+            "s_3_t_1_c_1_z_1_RGB.tiff",
```

Comment:
If we are going to change the "samples" dimension to the "channel" dimension I would rename this file to something like "s_1_t_1_c_(s)_3_z_1_RGB.tiff"... I think we all naturally assume `s_3...` is 3 scene.